packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -780,6 +807,24 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         }
     }
 
+    private renderMeasureSpan(id: string, displayString: string): JSX.Element {
+        return (
+            <span id={id} aria-hidden="true" className={this.props.managedClasses.site_measureSpan}>
+                {displayString}
+            </span>
+        );
+    }
+
+    private visualLength(id: string, ref: React.RefObject<HTMLSelectElement>): number {
+        const ruler: HTMLElement = document.getElementById(id);
```

Comment:
I'd recommend working inside of React's ecosystem and using refs for the rulers also - avoids any possible complication with update cycles and missing elements.